.github/workflows/backend.yml

```
@@ -111,3 +111,26 @@ jobs:
           node-version: ${{ env.NODE_VERSION }}
 
       - run: npm audit --audit-level high
+
+  vuln:
+    name: Run Trivy vulnerability scanner
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        service: [user, listing, review, message, lib]
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Build Docker image
+        run: |
+          docker compose up -d ${{ matrix.service }}
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.20.0
+        with:
+          image-ref: ${{ matrix.service }}
```

After we build the images, this is what they'd be
```suggestion
          image-ref: "martletplace-${{ matrix.service }}"
```